src/test/data/JsonSerializableClientBookTest/duplicateClientClientBook.json

```diff
@@ -1,11 +1,13 @@
 {
   "clients": [ {
+    "@UUID" : "c4fa6f3d-a49f-495d-a5eb-d282b344a36d",
     "name": "Alice Pauline",
     "phone": "94351253",
     "email": "alice@example.com",
     "address": "123, Jurong West Ave 6, #08-111",
     "tagged": [ "friends" ]
```

Comment:
These are missing the `meeting` field. Not sure if they will affect test runs using this json (unlikely to for now, but note for the future)